docs/main/introduction-to-openebs/introduction-to-openebs.md

```diff
@@ -34,16 +34,16 @@ OpenEBS manages the storage available on each of the Kubernetes nodes and uses t
 
 In case of [Local Volumes](#local-volumes):
 
-- OpenEBS can create persistent volumes using raw block devices or partitions, or using sub-directories on Hostpaths or by using local engine or sparse files.
+- OpenEBS can create persistent volumes using raw block devices or partitions, or using sub-directories on Hostpaths or by using Local Storage or sparse files.
 - The local volumes are directly mounted into the Stateful Pod, without any added overhead from OpenEBS in the data path, decreasing latency.
-- OpenEBS provides additional tooling for local volumes for monitoring, backup/restore, disaster recovery, snapshots when backed by local engine, capacity based scheduling, and more.
+- OpenEBS provides additional tooling for local volumes for monitoring, backup/restore, disaster recovery, snapshots when backed by Local Storage, capacity based scheduling, and more.
 
 In case of [Distributed (aka Replicated) Volumes](#replicated-volumes):
```

Comment:
Remove Distributed from all places. Distributed and Replicated are not the same.